etc/kayobe/kolla/inventory/group_vars/prometheus-blackbox-exporter

```diff
@@ -9,7 +9,7 @@
 prometheus_blackbox_exporter_endpoints: >-
   {{ (prometheus_blackbox_exporter_endpoints_kayobe | default([]) +
   prometheus_blackbox_exporter_endpoints_default) |
-  selectattr('enabled', 'true') |
+  selectattr('enabled', 'match', 'true|True') |
```

Comment:
Would this ignore any that use a proper boolean true, or "yes"?

Reply:
Rewritten this so it now captures all truthy values correctly. It follows the same format/convention @Alex-Welsh has in his firewalld config PR